docs/pt/docs/learn/index.md

```diff
@@ -0,0 +1,5 @@
+# Aprender
+
+Nesta parte da documentação encontramos as seções introdutórias e os tutoriais para aprendermos **FastAPI**.
```

Comment:
I will do it.

Reply:
Assuming it's a question, I don't think I've ever head it being treated as a feminine noun on the Brazilian side of the internet.

Reply:
@codespearhead, you assumed that this translation is just to serve Brazilian, but the name of its folder is not `pt-br` but `pt`.

Reply:
The reason for having only one folder for each language, such as English, Spanish, French, and Portuguese, as opposed to one for each dialect, is simply because the current folders already contain a substantial amount of markup written in a specific dialects and the community tends to prioritize completeness over fragmentation, especially because consolidating efforts towards the same goal leads to better outcomes.

However, I hear your frustration and I completely understand where you're coming from. If you prefer, I can drop the review.